client/src/endpoint/trade.rs

```diff
@@ -382,6 +382,36 @@ pub struct Aggregations {
 impl_limit!(Aggregations);
 impl_order!(Aggregations);
 
+/// The segment resolution used when querying for trade aggregations.
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum SegmentResolution {
+    /// Represents a one minute segment.
+    OneMin,
+    /// Represents a five minute segment.
+    FiveMin,
+    /// Represents a fifteen minute segment.
+    FifteenMin,
+    /// Represents an hour long segment.
+    OneHour,
+    /// Represents a day long segment.
+    OneDay,
+    /// Respresents a week long segment.
```

Comment:
These comments feel a bit superfluous.

Reply:
Yes, but required for `#[deny(missing_doc)]`